### PR TITLE
Homework 3

### PR DESCRIPTION
**Pull request by artsiomshevel**
homework 3 - ready for code review

### CODE REVIEW

File: Homework_3/homework_3.js [hunk 1]
```diff
@@ -1 +1,117 @@
...
+
+// Task2*. Решить квадратные уравнения. Переменные называть по правилам.
+// Вывести в консоль ответы в виде "Ответ к уравнению 1: <корень>", "Ответ к уравнению 2: <корень> и <корень>"
+// 1.  x2 - 6x + 9 = 0. - один корень
+let a = 1
+let b = -6
+let c = 9
+
+let d = b**2 - 4*a*c
+let x = -1*b / 2 * a
```
Comment (VladimirLQA) [inline Homework_3/homework_3.js]: Use `const` for values that never reassigned

**Review by VladimirLQA — CHANGES_REQUESTED** [review]

File: Homework_3/homework_3.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+// 18. Создать переменную item_6_type
+let item_6_type
+
+// 19. Присвоить переменной item_6 значение 15
+item_6 = 15
+console.log('item_6 = '+item_6)
+console.log(typeof(item_6))
+
+// 20. Присвоить переменной item_6_type тип переменной item_6
+item_6_type = 0
```
Comment (VladimirLQA) [inline Homework_3/homework_3.js]: You assigned `0` to `item_6_type`, not the type of `item_6`

File: Homework_3/homework_3.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+let x1 = (-1*b1 - Math.sqrt(d1)) / 2 * a1
+console.log(x1)
+let x2 = (-1*b1 + Math.sqrt(d1)) / 2 * a1
+console.log(x2)
+console.log('Ответ к уравнению 2: ' + x1 +' и ' + x2)
+
+// Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную 
+// n + nn + nnn, где n не перемножаются, а конкатенируются
+let n = 2
+let aa = n
```
Comment (VladimirLQA) [inline Homework_3/homework_3.js]: Unnecessary assigning for `n` variable, can be simplified

File: Homework_3/homework_3.js (hunk 1 above)
Comment (artsiomshevel) [reply Homework_3/homework_3.js]: updated @VladimirLQA

File: Homework_3/homework_3.js (hunk 2 above)
Comment (artsiomshevel) [reply Homework_3/homework_3.js]: updated @VladimirLQA

File: Homework_3/homework_3.js (hunk 3 above)
Comment (artsiomshevel) [reply Homework_3/homework_3.js:111]: updated @VladimirLQA

Comment (artsiomshevel) [reply Homework_3/homework_3.js]: updated @VladimirLQA

File: Homework_3/homework_3.js [hunk 4]
```diff
@@ -59,7 +59,7 @@ console.log('item_6 = '+item_6)
 console.log(typeof(item_6))
 
 // 20. Присвоить переменной item_6_type тип переменной item_6
-item_6_type = 0
+item_6_type = Number(item_6_type)
```
Comment (VladimirLQA) [inline Homework_3/homework_3.js]: You declared a variable `item_6_type` on 54 line, here you convert this variable that hasn't been assigned yet, pay attention.
If variable hasn't been assigned it gets an `undefiend` and what we get with this conversion `Number(undefiend)` ? 

On 63 line you logged a `typeof(item_6_type)` and looks like got an result, but try to console only the `item_6_type` you'll see :)

You have done it on 59 line in proper way

**Review by VladimirLQA — CHANGES_REQUESTED** [review]

File: Homework_3/homework_3.js [hunk 5]
```diff
@@ -107,11 +107,17 @@ console.log('Ответ к уравнению 2: ' + x1 +' и ' + x2)
 
 // Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную 
 // n + nn + nnn, где n не перемножаются, а конкатенируются
-let n = 2
+
+let n = 1
 let aa = n
```
Comment (VladimirLQA) [inline Homework_3/homework_3.js:112]: By "simplify" I mean that you can assign `1` to `aa` and use it.

File: Homework_3/homework_3.js [hunk 6]
```diff
@@ -1 +1,117 @@
...
+// 23. Создать переменную item_7_type
+let item_7_type
+
+// 24. Присвоить переменной item_7_type тип переменной item_7
+item_7_type = ''
+console.log(typeof(item_7_type))
+
+// 25. Вывести в консоль тип данных item_7 в виде ——  “item_7 == ”  item_7,  “item_7_type == ”  item_7_type ——  
+console.log('item_7 == ' + typeof(item_7) + ' ' + item_7)
+console.log('item_7_type == ' + typeof(item_7_type) + ' ' + item_7_type)
```
Comment (VladimirLQA) [inline Homework_3/homework_3.js]: Here same case as the previous
* 77 line - assigned empty string to item_7_type, but need to assign a type of item_7 (you've done it on 78 line)
* 81, 82 - you don't need to get types from variables `item_7 & item_7_type`

File: Homework_3/homework_3.js [hunk 7]
```diff
@@ -1 +1,117 @@
...
+// 20. Присвоить переменной item_6_type тип переменной item_6
+item_6_type = 0
+console.log(typeof(item_6_type))
+
+// 21. Вывести в консоль тип данных item_6 в виде ——  “item_6 == ”  item_6,  “item_6_type == ”  item_6_type ——  
+console.log('item_6 == ' + typeof(item_6) + ' ' + item_6)
+console.log('item_6_type == ' + typeof(item_6_type) + ' ' + item_6_type)
+
+// 22. Создать переменную item_7 и в ней преобразовать item_6 в String.
+let item_7 = item_6 + 'forString'
```
Comment (VladimirLQA) [inline Homework_3/homework_3.js]: 66, 67 - you don't need to get types from variables item_6 & item_6_type

Comment (VladimirLQA) [inline Homework_3/homework_3.js]: * 70 - convert to string without adding additional text

File: Homework_3/homework_3.js (hunk 4 above)
Comment (artsiomshevel) [reply Homework_3/homework_3.js]: the task is 'Вывести в консоль тип данных item_6 в виде ...'
and it is shown as 'item_6_type == number NaN'
'Тип данных' is displayed as I see :) @VladimirLQA

File: Homework_3/homework_3.js (hunk 6 above)
Comment (artsiomshevel) [reply Homework_3/homework_3.js]: updated @VladimirLQA

File: Homework_3/homework_3.js (hunk 4 above)
Comment (artsiomshevel) [reply Homework_3/homework_3.js]: updated @VladimirLQA

File: Homework_3/homework_3.js (hunk 7 above)
Comment (artsiomshevel) [reply Homework_3/homework_3.js]: updated @VladimirLQA

Comment (artsiomshevel) [reply Homework_3/homework_3.js]: updated @VladimirLQA

File: Homework_3/homework_3.js (hunk 5 above)
Comment (artsiomshevel) [reply Homework_3/homework_3.js:112]: @VladimirLQA but aa=n because n is checked in 'if' section

File: Homework_3/homework_3.js (hunk 4 above)
Comment (VladimirLQA) [reply Homework_3/homework_3.js]: As the result you should get this
```js
Item_6 = 15 | Item_6_type = number 
```
And you mentioned it `'item_6_type == number NaN'` - why did you get `NaN` ?

**Review by VladimirLQA — APPROVED** [review]

**Pull request merged**